packages/bot/src/stores/run-panel-store.js

```diff
@@ -263,6 +279,14 @@ export default class RunPanelStore {
         } else {
             this.error_type = error_types.RECOVERABLE_ERRORS;
         }
+        
+        this.ShowErrorMessage(data);
```

Comment:
This is capitalised, should it be like below?

```suggestion
        this.showErrorMessage(data);
```